runtime/tcpsrv.h

```diff
@@ -177,6 +178,8 @@ BEGINinterface(tcpsrv) /* name must also be changed in ENDinterface macro! */
 	rsRetVal (*SetbSPFramingFix)(tcpsrv_t*, sbool);
 	/* added v19 -- PascalWithopf, 2017-08-08 */
 	rsRetVal (*SetGnutlsPriorityString)(tcpsrv_t*, uchar*);
+	/* added v?? -- Preserve case in fromhost, 2018-07-?? */
+	rsRetVal (*SetPreserveCase)(tcpsrv_t *pThis, int bPreserveCase);
 ENDinterface(tcpsrv)
 #define tcpsrvCURR_IF_VERSION 20 /* increment whenever you change the interface structure! */
```

Comment:
while this interface is no longer really enforced, the IF_VERSION should be incremented on interface change. So 21 would be the correct value now.

Reply:
Thank you, @rgerhards!  I've updated tcpsrv.h and rebased.